src/Features/Core/Portable/Completion/CompletionOptions.cs

```diff
@@ -17,5 +17,9 @@ internal static class CompletionOptions
         public static readonly Option<bool> AlwaysShowBuilder = new Option<bool>(ControllerFeatureName, "AlwaysShowBuilder", defaultValue: false);
         public static readonly Option<bool> FilterOutOfScopeLocals = new Option<bool>(ControllerFeatureName, "FilterOutOfScopeLocals", defaultValue: true);
         public static readonly Option<bool> ShowXmlDocCommentCompletion = new Option<bool>(ControllerFeatureName, "ShowXmlDocCommentCompletion", defaultValue: true);
+
+        // Dev15 options
+        public static readonly PerLanguageOption<bool> ShowCompletionItemFilters = new PerLanguageOption<bool>(FeatureName, nameof(ShowCompletionItemFilters), defaultValue: false);
```

Comment:
@jasonmalinowski is this okay?  I thought we couldn't mix `PerLanguageOption` and regular `Option`.